utils/python/CIME/SystemTests/tests/case_fake.py

```diff
@@ -0,0 +1,99 @@
+"""
+This module contains a fake implementation of the Case class that can be used
+for testing the tests.
+"""
+
+import os
+from copy import deepcopy
+
+class CaseFake(object):
```

Comment:
If this CaseFake class would have more general utility, it could be moved to some higher level